frog/imports/client/GraphEditor/Graph.js

```diff
@@ -22,8 +22,8 @@ const scrollMouse = e => {
 };
 
 const mousemove = e => {
-  // We do -100 here because there is 100px above the graph editor
-  store.ui.socialMove(e.clientX, e.clientY - 100);
+  // Subtracting Graph Editor's position from the top of the screen
```

Comment:
This should not be part of this PR.

Reply:
Ah yeah I branched this by mistake. I'll fix this.